modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -124,17 +125,15 @@ def number_and_street
     end
 
     def city
-      veteran_data.dig('address', 'city') || ''
+      veteran.city || ''
```

Comment:
I updated all three of these methods to rely on the `veteran` instance of `Appellant` instead of using the duplicated (vs. `Appellant`) `dig` logic